cscs-checks/prgenv/helloworld.py

```diff
@@ -75,6 +83,9 @@ def num_ranks(match):
                 ),
             )
         )
+
+    @run_after('init')
```

Comment:
I think we could try and get the hooks in pipeline-order when possible.
```suggestion
    @run_before('performance')
```